gapic/api/artman_common.yaml

```diff
@@ -0,0 +1,25 @@
+common:
```

Comment:
execute_pipeline.py is a generic utility and does not have good commandline to allow expanding variables like 'API_VERSION' or 'API_NAME'. My new commandline tool, api_codegen.py, has narrower focus and therefore can use this simplified format.

Until execute_pipeline.py is redesigned to allow this config, I think it's better to keep the individual API config files.